### DISCUSSION TITLE
kubernetes-handbook/concepts/network-policy/

### DISCUSSION THREAD

**Discussion by giscus** [rootsongjc/cloud-native-library#20] [Announcements]
# kubernetes-handbook/concepts/network-policy/

网络策略说明一组 Pod 之间是如何被允许互相通信，以及如何与其它网络 Endpoint 进行通信。 NetworkPolicy 资源使用标签来选择 Pod，并定义了一些规则，这些规则指明允许什么流量进入到选中的 Pod 上。关于 Network Policy 的详细用法请参考 Kubernetes 官网。 Network Policy 的

https://lib.jimmysong.io/kubernetes-handbook/concepts/network-policy/

**Comment by rootsongjc** [comment]
Network Policy 是作用在 L3/4 层的，也就是说只能通过标签选择器选择出一些 Pod，限制这些 Pod 对某些网段和端口的访问，如果有从应用层来限制网络，可以使用 Istio 或者 Cilium。
